oss-cloud-backend/handler.js

```diff
@@ -13,3 +15,28 @@ module.exports.hello = async event => {
     )
   };
 };
+
+
+// returns all pull requests from parents of all forked repos for a single github user
+// POST expected json
+// {
+//    username: string  
+// }
+module.exports.getPulls = (event, context, callback) => {
+  var body = JSON.parse(event.body);
+  getPullsHandler.getForkedRepos(body.username) // get all repos of a user
```

Comment:
I would propose to use async/await syntax to deal with promises. It's more clean and readable.

Reply:
Switched syntax.